codestodians/komlie-api
# [Feature] Add quantity in stock to product model

#### What does this PR do?
- Adds the `quantity_in_stock` field to the product model.

#### Description of Tasks completed?
- Add the `quantity_in_stock` field to the product model.
- Create migration for the added field
- Refactor affected schema  (`product` and `cart`)
- Update tests

#### How should this be manually tested?
- Checkout into the `feature/add-quantity-in-stock-to-product-model-167425339` branch.
- Run the latest migration
- Start the server and make mutations as necessary to confirm everything is in order
- Run the tests

#### Any background context you want to provide?
- The idea is to prevent addition to cart of more items than available.

#### What are the relevant pivotal tracker stories?
- [#167425339](https://www.pivotaltracker.com/story/show/167425339)
